docs/usage.md

```diff
@@ -43,15 +46,13 @@ The following is an ideal flow to use the **Outbreak Response Framework** soluti
 
         ![Signature Based Threat Hunting Rules](./res/sigma_rule.png)
 
-8. **Ticketing/ITSM**: If any vulnerability is found using the IOC Threat Hunt or Signature Based Threat Hunt automatically Alert is created for that vulnerability in FortiSOAR and as part of your response or threat management strategy automatically creates tickets using Ticketing/ITSM Integrations (Jira/ServiceNow) for Outbreak Alert(vulnerability). This step is performed only if you have added a ticketing/ITSM solution in the configuration wizard.
-
-9. **Remediation**: We can take remediation using two ways
+8. **Remediation**: We can take remediation using two ways
```

Comment:
This part have been removed now. We dnt use fortigate anymore to block the IOCs.